### ISSUE TITLE
link/@rel='convertedFrom" mandatory?

### ISSUE THREAD

**Issue by reschke** [rfc-format/draft-iab-rfcv3-preptool-bis#3]
I'd like to challenge this requirement from https://greenbytes.de/tech/webdav/rfc7998.html#rfc.section.5.6.3:

> 4. If in RFC production mode, check if there is a <link> element with the file name of the Internet-Draft that became this RFC the form <link rel="convertedFrom" href="https://datatracker.ietf.org/doc/draft-tttttttttt/">. If one does not exist, give an error.

For example, for RFC 7998, the link would be to 

  https://datatracker.ietf.org/doc/draft-iab-rfcv3-preptool/

Following this link however leads to a 302 redirect to

  https://datatracker.ietf.org/doc/rfc7998/

at which point it becomes very unclear why we bother with that, if all we do is linking to the data tracker, and the data tracker already has all the information.

**Comment by hildjj** [comment]
Agree.  There are also cases where no I-D ever existed, such as April1 RFCs.  Warning?  Nothing?

**Comment by reschke** [comment]
Remove the link mechanism? Why do we need it?

**Comment by hildjj** [comment]
We had a long discussion about that on the mailing list, didn't we?  @paulehoffman do you remember why we ended up here?

**Comment by reschke** [comment]
It would be great if we could make progress on these tickets. I believe a 3rd implementation of the prep tool mechanism is in the works, and it would be good if we could avoid implementing stuff that'll get removed later anyway.
